app/Http/Controllers/Auth/AuthController.php

```diff
@@ -179,8 +185,55 @@ public function postOauth(Request $request)
         return redirect($this->redirectPath());
     }
 
+    /**
+     * Show form to collect new username.
+     * 
+     * @return Response
+     */
     public function getOauth()
     {
         return view('/errors/oauthname');
     }
+
+    /**
+     * Send a mail to register a user.
+     * 
+     * @param  Request $request
+     * @return Response
+     */
+    public function sendMail(Request $request)
+    {
+        $validator = $this->validator($request->all());
+
+        if ($validator->fails()) {
+            $this->throwValidationException(
+                $request, $validator
```

Comment:
Okay, I'll change it. Laravel code has been otherwise.


On Mon, Nov 23, 2015 at 12:16 PM, Prosper Otemuyiwa <
notifications@github.com> wrote:

> In app/Http/Controllers/Auth/AuthController.php
> <https://github.com/andela/pibbble/pull/141#discussion_r45592262>:
>
> >      public function getOauth()
> >      {
> >          return view('/errors/oauthname');
> >      }
> > +
> > +    /**
> > +     * Send a mail to register a user.
> > +     *
> > +     * @param  Request $request
> > +     * @return Response
> > +     */
> > +    public function sendMail(Request $request)
> > +    {
> > +        $validator = $this->validator($request->all());
> > +
> > +        if ($validator->fails()) {
> > +            $this->throwValidationException(
> > +                $request, $validator
>
> This means it should be like so:
>
> if ($validator->fails()) {   $this->throwValidationException(       $request,       $validator  );}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/pibbble/pull/141/files#r45592262>.
>



-- 
*Opeyemi Ogunjimi | Software Developer | Andela*
(c) +234-703-936-2475 | Andela.com <http://andela.com/>